index.html

```diff
@@ -59,7 +59,7 @@
             <script type="text/javascript" src="./scripts/static_text/siteheader/content_siteheader_table0_row0.js">
             </script>
             <!--v   Open Row0 Column0   v-->
```

Comment:
(Review comment 2) What are your thoughts on the HTML comments? If they don't add insight, we can consider removing. The original thought was that it will help identify bugs in the HTML, because we reference external JS files (the reference needs to match the name of the JS file, the name of the JS file needs to match the right location in the table, and the JS inside the file needs to reference the right ID in the HTML file. There seemed to be enough moving parts here to add the comments that hopefully make debugging a bit easier.